src/utils/test/stringFormat.test.ts

```diff
@@ -10,3 +11,27 @@ test('getDateString tests', () => {
   expect(getDateString(new Date(2020, 5, 10))).toBe('6/10/2020');
   expect(getDateString(new Date(2025, 11, 21))).toBe('12/21/2025');
 });
+
+// Set up dict
+const shortHandNames: { [fullName: string]: string } = {
```

Comment:
💭 Could maybe just export this as a constant somewhere (assets, maybe) and import here and in the map